molecule/config.py

```diff
@@ -94,13 +94,15 @@ def build_easy_paths(self):
 
         :return: None
         """
-        self.config['molecule']['state_file'] = os.path.join(self.config['molecule']['molecule_dir'],
-                                                             self.config['molecule']['state_file'])
-        self.config['molecule']['vagrantfile_file'] = os.path.join(self.config['molecule']['molecule_dir'],
-                                                                   self.config['molecule']['vagrantfile_file'])
-        self.config['molecule']['rakefile_file'] = os.path.join(self.config['molecule']['molecule_dir'],
-                                                                self.config['molecule']['rakefile_file'])
-        self.config['ansible']['config_file'] = os.path.join(self.config['molecule']['molecule_dir'],
-                                                             self.config['ansible']['config_file'])
-        self.config['ansible']['inventory_file'] = os.path.join(self.config['molecule']['molecule_dir'],
-                                                                self.config['ansible']['inventory_file'])
+        vals = [
+            ('molecule', 'state_file'),
+            ('molecule', 'vagrantfile_file'),
+            ('molecule', 'rakefile_file'),
+            ('ansible', 'config_file'),
+            ('ansible', 'inventory_file')
+        ]
+
+        for k, v in vals:
```

Comment:
In that case it is not ``key`` and ``value`` anymore, so I would replace ``k`` and ``v`` with more meaningful variable names.